maps/us-healthcare-system-capacity/js/main.js

```diff
@@ -282,7 +288,14 @@ function resetFillPaintStyle(layerName) {
 }
 
 function usePerCapita(theIndicator) {
-  return number !== 0 && theIndicator !== 3 && theIndicator !== 4 && type !== 3;
+  // number == 0 is showing total values
+  // Indicators 3 and 4 are capacity
+  // type == 3 is facility-level data
+  return number != 0 && !(theIndicator == 3 || theIndicator == 4) && type !== 3;
```

Comment:
Using `number !== 0` was causing this not to work correctly (this is why `undefined` was showing up)